src/app/frontend/commons/commons_module.js

```diff
@@ -0,0 +1,11 @@
+import errorDialogService from './errordialog_service';
+
```

Comment:
This looks good to me. Angular 2.0 and 1.5 also has strong connection with components/ directory so this would align us a little with it.